app/views/devise/registrations/edit.html.haml

```diff
@@ -11,31 +11,29 @@
       = f.email_field :email, autofocus: true, class: 'wpcf7-form-control'
     - if devise_mapping.confirmable? && resource.pending_reconfirmation?
       %div
-        Obekräftad mailadress för: #{resource.unconfirmed_email}
+        "#{t('.unconfirmed_email_for', unconfirmed_email: resource.unconfirmed_email )}"
     .field
-      = f.label :password, 'Lösenord'
-      %i (lämna tomt om du inte vill byta)
+      = f.label :password
+      %i #{t('.leave_blank_if_no_change')}
```

Comment:
He suggests 
      `%i #{t('.leave_blank_if_no_change')}` 
to be 
      `%i= t('.leave_blank_if_no_change')`